feature/gribi/otg_tests/base_leader_election_test/base_leader_election_test.go

```diff
@@ -174,10 +175,9 @@ func configureATE(t *testing.T, ate *ondatra.ATEDevice) gosnappi.Config {
 
 func waitOTGARPEntry(t *testing.T) {
 	ate := ondatra.ATE(t, "ate")
-	ate.OTG().Telemetry().InterfaceAny().Ipv4NeighborAny().LinkLayerAddress().Watch(
-		t, time.Minute, func(val *otgtelemetry.QualifiedString) bool {
-			return val.IsPresent()
-		}).Await(t)
+	gnmi.WatchAll(t, ate.OTG(), gnmi.OTG().InterfaceAny().Ipv4NeighborAny().LinkLayerAddress().State(), time.Minute, func(val *ygnmi.Value[string]) bool {
+		return val.IsPresent()
+	}).Await(t)
```

Comment:
Does this actually do anything? If no value is present it'll timeout and return false but nothing reacts to it. Was this true of the test already?

Reply:
that's correct. the semantics of watch haven't changed, so that what the test was doing previously.